src/App.jsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 
 const App = () => {
   return (
-    <>
+    <div data-cy="siteName">
```

Comment:
Please get into the habit of using `kebab-case` for these data attributes. That is the practice that we use here at Craft and the one that is most widely used in the industry as far as we are concerned.

Reply:
we will remember to do that! but for now we just deleted the data-cy because we deleted the test for it :)